pydantic/generics.py

```diff
@@ -140,6 +144,53 @@ def __concrete_name__(cls: Type[Any], params: Tuple[Type[Any], ...]) -> str:
         params_component = ', '.join(param_names)
         return f'{cls.__name__}[{params_component}]'
 
+    @classmethod
+    def __parameterized_bases__(cls, typevars_map: Mapping[TypeVarType, Type[Any]]) -> Iterator[Type[Any]]:
```

Comment:
`Mapping[TypeVarType, Type[Any]]`

could be aliased and reused

Reply:
aliased to `Parametrization`